.gitignore

```diff
@@ -46,11 +46,14 @@ external_deps/*
 !.gitattributes
 !.gitignore
 
-#ignore Visual Stuido generated project files
+#ignore Visual Studio generated project files
 *.vcxproj
 *.vcxproj.filters
 *.vcxproj.user
 *.sln
 
 #ignore Visual Studio / Visual Studio Code settings
 .vs
+
+#ignore downloaded recastnavigation lib
+libs/recastnavigation
```

Comment:
That gitignore addition is genuine but that comment does not exactly describes what happened.

In fact `libs/recastnavigation` was a submodule, but this submodule was removed from engine and added to the game repository (`Unvanquished`) instead because that code was moved from engine to game code, see 13829dac0e9b55489ea3347b801f4bedac72ff11

So, yes we have to ignore this line for people who try to build 0.52 after having built 0.51, maybe the comment can be something like:

```
#ignore removed recastnavigation submodule
```

Reply:
Fixed as suggested, thanks!

Reply:
well, I was just giving a quick eye on the patch... thank you to improve the game I like to play to!